### ISSUE TITLE
Browser: 'chrome' was not found on your system or is not supported by Cypress.

### ISSUE THREAD

**Issue by MaximeCheramy** [cypress-io/cypress#8736]
### Current behavior

We are building a docker image with cypress, chrome and all the spec files (using cypress/included). This image is run from our instance of Gitlab CI automatically. Only the `CYPRESS_BASE_URL` environment variable and the name of the spec file are passed to the docker run command line.

Sometimes, very rarely, we have the following error:

```
> cypress run --headless --browser chrome --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json "--spec" "cypress/integration/interview.spec.ts"
[15:37:14]  Verifying Cypress can run /root/.cache/Cypress/5.3.0/Cypress [started]
[15:37:16]  Verifying Cypress can run /root/.cache/Cypress/5.3.0/Cypress [completed]
Missing baseUrl in compilerOptions. tsconfig-paths will be skipped
Couldn't determine Mocha version
Can't run because you've entered an invalid browser name.
Browser: 'chrome' was not found on your system or is not supported by Cypress.
Cypress supports the following browsers:
- chrome
- chromium
- edge
- electron
- firefox
You can also use a custom browser: https://on.cypress.io/customize-browsers
Available browsers found on your system are:
- firefox
- electron
```

The system automatically retry failed jobs and it works the second time.

### Desired behavior

Not crash.

### Versions

Cypress 5.3.0 inside a Docker running on a Ubuntu 20.04.

**Comment by scottcrowe** [comment]
Seeing the same inconsistent behavior in a image built off of the `cypress/browsers:node10.16.0-chrome77`

**Comment by phao5814** [comment]
Just thought that I'd reiterate that I'm experiencing the same issue using the **official** cypress Docker image, specifically "cypress/included:5.3.0".

The command I'm running is as follows:

`NODE_ENV=test $(npm bin)/cypress run --config video=false --browser chrome --headless --record --key <my_key> --parallel --ci-build-id $BUILD_NUMBER`

Below is the error I'm receiving. Like the others, it happens very inconsistently.

> cypress_1  | Can't run because you've entered an invalid browser name.
> cypress_1  | Browser: 'chrome' was not found on your system or is not supported by Cypress.
> cypress_1  | Cypress supports the following browsers:
> cypress_1  | - chrome
> cypress_1  | - chromium
> cypress_1  | - edge
> cypress_1  | - electron
> cypress_1  | - firefox
> cypress_1  | You can also use a custom browser: https://on.cypress.io/customize-browsers
> cypress_1  | Available browsers found on your system are:
> cypress_1  | - firefox
> cypress_1  | - electron
> 

The docker image is run by a Jenkins server running on Ubuntu 18.04.

**Comment by barakplasma** [comment]
I think this issue belongs on the https://github.com/cypress-io/cypress-docker-images instead of this repo

**Comment by rachelruderman** [comment]
I can confirm that this error also (rarely) happens on `cypress/included:6.3.0`

**Comment by jennifer-shehane** [comment]
Duplicate of https://github.com/cypress-io/cypress/issues/8611
